dev/com.ibm.ws.cdi.1.2.weld/src/com/ibm/ws/cdi/proxy/ProxyServicesImpl.java

```diff
@@ -132,5 +139,32 @@ protected void addWeldDynamicImports(Bundle b, ManifestElement[] dynamicImports)
             throw new RuntimeException(e.getTargetException());
         }
     }
+
+    private void stackCheck() {
```

Comment:
A javadoc comment for this method stating what it's intending to check would be helpful.

Maybe something like `checkCallerIsWeld()` would be a better name, but I'm not super concerned about that.